sdk/storage/storage-blob/test/node/appendblobclient.spec.ts

```diff
@@ -151,6 +153,102 @@ describe("AppendBlobClient Node.js only", () => {
     assert.equal(downloadResponse.contentLength!, content.length * 2);
   });
 
+  it("appendBlockFromURL - source SAS and destination bearer token", async function() {
+    if (!isPlaybackMode()) {
+      // Enable this when STG78 - version 2020-10-02 is enabled on production.
+      this.skip();
```

Comment:
open an issue to track the enabling

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/16208